libsweep/include/sweep/sweep.h

```diff
@@ -10,12 +10,28 @@
 extern "C" {
 #endif
 
+#if defined _WIN32 || defined __CYGWIN__ || defined __MINGW32__
+// If we are building a dll, set SWEEP_API to export symbols
+#ifdef BUILDING_DLL
```

Comment:
Isn't `BUILDING_DLL` the equivalent of `hairpoll_EXPORTS` on the slide you mentioned?